modules/ipi-install-assigning-a-static-ip-address-to-the-bootstrap-vm.adoc

```diff
@@ -0,0 +1,74 @@
+// This is included in the following assemblies:
+//
+// ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="assigning-a-static-ip-address-to-the-bootstrap-vm_{context}"]
+= Assigning a static IP address to the bootstrap VM
+
+If you are deploying {product-title} without a DHCP server on the `baremetal` network, you must configure a static IP address for the bootstrap VM using Ignition.
+
+.Procedure
+
+. Generate the manifests:
++
+[source,terminal]
+----
+$ ./openshift-baremetal-install --dir ~/clusterconfigs create manifests
```

Comment:
Creating manifests is already listed earlier on in the installation instructions.